src/main/java/mods/railcraft/common/plugins/ic2/IC2Plugin.java

```diff
@@ -143,6 +143,18 @@ public static void addMaceratorRecipe(@Nullable ItemStack input, @Nullable ItemS
         }
     }
 
+    public static void addCrushedOreMaceratorRecipe(@Nullable ItemStack input, int numinput, @Nullable ItemStack output) {
+        if (input == null || output == null)
+            return;
+        output = output.copy();
+        output.stackSize = 2;
```

Comment:
It's based off of the Rock Crusher function, but I could probably ignore that and make this more generic.